api/ls_ganap/main_events/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
```

Comment:
![screen shot 2018-08-06 at 2 49 10 pm](https://user-images.githubusercontent.com/20733806/43701240-09c0157e-9988-11e8-9e96-39a494179d64.png)

Deleted at should be visible, but should not be editable